### DISCUSSION TITLE
How to simulate 10Hz rTMS？

### DISCUSSION THREAD

**Discussion by xiongyy99255** [simnibs/simnibs#242] [8. Simulation workflows]
Hi SimNIBS team, thanks for the great tool!
     Is it possible to simulate 10Hz rTMS neurostimulation, i.e. generating time-varying electromagnetic fields on a time scale?

Thanks！

**Comment by diedrim** [comment]
At low frequencies of TMS the problem is quasi-static and can be separated into a spatial and a temporal component. SimNIBS only calculates the spatial component (https://simnibs.github.io/simnibs/build/html/faq.html), see first two issues in the FAQ, where the same topic is discussed for TACS and the TMS intensity scaling is described.
